src/seedu/addressbook/data/person/UniquePersonList.java

```diff
@@ -1,15 +1,10 @@
 package seedu.addressbook.data.person;
 
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Collection;
-import java.util.Collections;
-import java.util.Iterator;
-import java.util.List;
-
 import seedu.addressbook.common.Utils;
 import seedu.addressbook.data.exception.DuplicateDataException;
 
+import java.util.*;
```

Comment:
As mentioned by Jing, wildcard imports violate the coding standard. Please use single imports and change your settings in Intellij: `Settings` > `Editor` > `Code Style` > `Java` > `Imports` and set both class count and names count to 99 (or any large number)